plugins/chat/db/migrate/20241003122030_add_notification_level_to_user_chat_channel_memberships.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class AddNotificationLevelToUserChatChannelMemberships < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def up
+    execute <<~SQL
+      DROP INDEX CONCURRENTLY IF EXISTS user_chat_channel_memberships_index
+    SQL
+
+    add_column :user_chat_channel_memberships, :notification_level, :integer, default: 1
```

Comment:
Thank you for helping sanity check this, there is quite a lot happening with these migrations.

I was thinking about moving the index and reapplying in a separate migration, I think it can be applied safely after everything else is complete. The initial migration only needs to focus on adding the new column.

I'm wondering is there any benefit to using post migrations here?

Reply:
I was thinking it might have been the combo of setting both default value and null false that done it before but better not to take the risk. I have updated the steps and moved this into a single migration, let me know what you think.